### PR TITLE
ZCS-11866 : Install zimbra-rabbitmq packages

### PR DESCRIPTION
**Pull request by umagmrit**
Related PR's:
https://github.com/Zimbra/packages/pull/169
https://github.com/Zimbra/zm-core-utils/pull/123

### CODE REVIEW

**Review by dasiyogesh — APPROVED** [review]

**Review by preshitaagrawal — APPROVED** [review]

File: rpmconf/Install/Util/modules/install_rabbitmq.sh [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
```
Comment (silentsakky) [inline rpmconf/Install/Util/modules/install_rabbitmq.sh:1]: a little bit confused if https://github.com/Zimbra/zm-build/pull/228/files#diff-959e3a8de3363d97be718a30338f1e76e23fce4f4030da828aae140cc039a19cR2487 line is already installing the package then why do we need this file?

Comment (umagmrit) [reply rpmconf/Install/Util/modules/install_rabbitmq.sh]: not required, committed changes for the same

**Review by zimsuchitgupta — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

File: rpmconf/Install/Util/utilfunc.sh [hunk 2]
```diff
@@ -2484,7 +2484,7 @@ isOnlyofficeStandalone() {
   fi
   ## install rabbit mq
   if [ $onlyofficepkg == "yes" ]; then
-    INSTALL_PACKAGES="$INSTALL_PACKAGES rabbit-mq"
+    INSTALL_PACKAGES="$INSTALL_PACKAGES zimbra-rabbitmq-server"
```
Comment (silentsakky) [inline rpmconf/Install/Util/utilfunc.sh:2487]: Approved for now, but in future we should remove this change and add zimbra-rabbitmq-server as a dependency of zimbra-onlyoffice package so we don't have to install it separately

**Review by silentsakky — APPROVED** [review]

File: rpmconf/Install/Util/utilfunc.sh [hunk 3]
```diff
@@ -1734,6 +1730,16 @@ removeZextrasPackagesIfInstalled() {
 	done
 }
 
+removeExternalRabbitmqIfInstalled(){
+	isInstalled rabbitmq-server
```
Comment (jiteshsojitra) [inline rpmconf/Install/Util/utilfunc.sh:1734]: `isInstalled "rabbitmq-server"`

**Review by jiteshsojitra — APPROVED** [review]

**Comment by jiteshsojitra** [comment]
For future, may be like below:

image

```
buildDir="/home/build"
repoDir="${buildDir}/stash"
zimbraBinDir="opt/zimbra/bin"

repoName="zm-core-utils"
utilArray=(
	"zmrabbitmqctl"
	"zmvolume"
)
for util in "${utilArray[@]}"; do
	echo -e "Copying ${repoDir}/${zm-core-utils}/src/bin/${util} to ${repoDir}/zm-build/${currentPackage}/${zimbraBinDir}/${util}"
	Copy "${repoDir}/${zm-core-utils}/src/bin/${util}" "${repoDir}/zm-build/${currentPackage}/${zimbraBinDir}/${util}"
done
```

**Review by silentsakky — APPROVED** [review]

**Pull request merged**